README.md

```diff
@@ -1,16 +1,16 @@
-# Machine Learning for the Elastic Stack
+"# Machine Learning for the Elastic Stac2
```

Comment:
`Stac2` ?

Reply:
Thanks.  Also there's the `"` at the beginning.  It's due to https://github.com/vim/vim/issues/2238.  Unless there's a workaround I might have to stop using `vim`!